data/quarg/quarg news.txt

```diff
@@ -442,3 +442,65 @@ news "student on quarg planets"
 			"."
 		word
 			`"`
+
+news "quarg anywhere"
+	location
+		government "Quarg"
```

Comment:
Would this not trigger in regions where quarg don't speak human languages well, or at all?

Reply:
You probably need to do regional filtering, using the `human quarg`, `hai quarg` attributes. (iirc those two speak human languages well)

Also, what about Humanika? I never know which filters account for it.

Reply:
Humanika has `farming human south` for its attributes.
Given the Quarg basically solely interact there via the robots, I don't think it should be gettin the news as an exception to the attributes.